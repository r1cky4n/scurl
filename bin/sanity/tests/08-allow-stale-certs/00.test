--allow-stale-certs 10000 https://expired.badssl.com/
