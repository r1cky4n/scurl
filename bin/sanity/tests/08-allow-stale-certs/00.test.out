<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="/icons/favicon-red.ico"/>
  <link rel="apple-touch-icon" href="/icons/icon-red.png"/>
  <title>expired.badssl.com</title>
  <link rel="stylesheet" href="/style.css">
  <style>body { background: red; }</style>
</head>
<body>
<div id="content">
  <h1 style="font-size: 12vw;">
    expired.<br>badssl.com
  </h1>
</div>

</body>
</html>
